src/applications/ivc-champva/10-10D/components/File/FileUpload.jsx

```diff
@@ -25,11 +41,15 @@ export function FileFieldCustom(props) {
         uiSchema={props.uiSchema}
         name={props.name}
         title={props.title}
-        data={props.data}
         pagePerItemIndex={props.pagePerItemIndex}
+        data={
+          props.pagePerItemIndex !== undefined && props.onReviewPage
+            ? props.data.applicants[props.pagePerItemIndex]
+            : props.data
+        }
```

Comment:
Doing this so that we get the right data passed to the custom page if we're on a list loop page or outside of the loop.

Reply:
So, if outside the list loop, `data` will just be all the form data like normal. Inside a list loop, we just pass the specific applicant's data.